babel.config.json

```diff
@@ -1,8 +1,8 @@
 {
   "presets": ["@babel/preset-env"],
   "plugins": [
-    "@babel/plugin-transform-object-assign",
-    "@babel/plugin-proposal-class-properties"
+    "@babel/plugin-proposal-class-properties",
+    "@babel/plugin-transform-object-assign"
```

Comment:
nit: order only changed

Reply:
yeah, I had it this way in this PR, but one of the earlier ones got in first with a different order. I sort of preferred this order though because it's alphabetical